include/util/guidance/turn_lanes.hpp

```diff
@@ -27,6 +28,11 @@ template <> struct hash<::osrm::util::guidance::LaneTuple>
 {
     inline std::size_t operator()(const ::osrm::util::guidance::LaneTuple &bearing_class) const;
 };
+template <> struct hash<::osrm::util::guidance::LaneTupleIdPair>
+{
+    inline std::size_t
+    operator()(const ::osrm::util::guidance::LaneTupleIdPair &bearing_class) const;
+};
```

Comment:
Only a declaration? This looks like dead code? You seem to implement the hash_value function below?